content/03.results.md

```diff
@@ -16,10 +16,10 @@ The Portal contains data from 700 samples and 55 tumor types [@doi:10.1016/j.dev
 <!-- TODO: Update numbers -->
 Figure {@fig:fig1}A summarizes all samples from patient tumors and patient-derived xenografts currently available on the Portal.
 The total number of samples for each diagnosis is shown, along with the proportion of samples from each disease stage within a diagnosis group.
-The largest number of samples found on the Portal were obtained from patients with leukemia (n = 191).
-The Portal also includes samples from brain and central nervous system tumors (n = 166), sarcoma and soft tissue tumors (n = 68), and a variety of other solid tumors (n = 86).
-Most samples were collected at initial diagnosis (n = 426), with a smaller number of samples collected either at recurrence (n = 67), during progressive disease (n = 12), or post-mortem (n = 5).
-Along with the patient tumors, the Portal contains a small number of human tumor cell line samples (n = 4).
+The largest number of samples found on the Portal were obtained from patients with leukemia (n = 216).
+The Portal also includes samples from sarcoma and soft tissue tumors (n = 194), brain and central nervous system tumors (n = 167), and a variety of other solid tumors (n = 117).
+Most samples were collected at initial diagnosis (n = 521), with a smaller number of samples collected either at recurrence (n = 129), during progressive disease (n = 13), during or after treatment (n = 11), or post-mortem (n = 5).
+Along with the patient tumors, the Portal contains a small number of human tumor cell line samples (n = 6).
```

Comment:
So, only the 4 NB cell lines are marked as cell lines on the Portal. The PDX lines (https://scpca.alexslemonade.org/projects/SCPCP000024) are marked as PDX. I see your point here, but we should either be consistent with the Portal or perhaps set `is_cell_line` and `is_xenograft` both to `TRUE`.

Reply:
Ah okay I was confused about this because I thought we had 6 cell lines, but then when I looked at the portal there was only 4 and then I filtered our sample metadata by `is_cell_line` and only saw 4. I think that they should both be set to true.

Reply:
If you're good with that, then I'll file an issue to update that in `ScPCA-admin` and group that with the other metadata changes for hand off and then update the number here to 6.

Reply:
Yea, that's fine. In practice, we should bundle updating this with the other data release 🎟️

Reply:
You said that already. I missed it.